features/User.feature

```diff
@@ -25,4 +25,22 @@ Feature: User Commands
       | profileIds | ["admin"]           |
       | email      | "alyx@blackmesa.us" |
 
+  # user:export-mapping & user:import-mapping ==================================================
 
+  @security
+  Scenario: Export and import users mappings
```

Comment:
No need to add again the plural form, user mappings; can mean multiple users and/or multiple mappings.
```suggestion
  Scenario: Export and import user mappings
```